lambda/deploy_function.yaml

```diff
@@ -0,0 +1,29 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
AWS lambda isn't needed. ADFS server provides human friendly account names. They are already used to group aws roles within an account.